charts/kong/templates/_helpers.tpl

```diff
@@ -166,6 +167,7 @@ spec:
   - name: kong-{{ .serviceName }}-tls
     port: {{ .tls.servicePort }}
     targetPort: {{ .tls.overrideServiceTargetPort | default .tls.containerPort }}
+    appProtocol: tls
```

Comment:
```suggestion
    appProtocol: https
```

Reply:
Sure. By the way, this is actually the same for Istio, I just wanted to make it consistent with the existing identifiers

> TLS Encrypted data. Because the Sidecar does not decrypt TLS traffic, this is the same as tls